samples/openapi3/client/petstore/python-nextgen-aiohttp/petstore_api/models/mixed_properties_and_additional_properties_class.py

```diff
@@ -76,7 +76,7 @@ def from_dict(cls, obj: dict) -> MixedPropertiesAndAdditionalPropertiesClass:
         _obj = MixedPropertiesAndAdditionalPropertiesClass.parse_obj({
             "uuid": obj.get("uuid"),
             "date_time": obj.get("dateTime"),
-            "map": dict((_k, Dict[str, Animal].from_dict(_v)) for _k, _v in obj.get("map").items())
+            "map": dict((_k, Dict[str, Animal].from_dict(_v)) for _k, _v in obj.get("map").items()) if obj.get("map") is not None else None
```

Comment:
Thanks for the fix!

This still has the problem that `Dict[str, Animal].from_dict(_v)` will fail. Should I open a separate bug report for this?

Reply:
no need. i'll try to fix it in another PR

Reply:
Filed https://github.com/OpenAPITools/openapi-generator/pull/15018 to fix it. Please review when you've time.